.coderabbit.yaml

```
@@ -0,0 +1,61 @@
+ignore
+  - Art
+  - Prefabs
+  - Scenes
+  - ThirdParty
+  - Tests
+
+# Messages personnalisés
+messages
+  welcome 🚀 CodeRabbit prêt.
+  closing ✅ Review terminée, tié un tigre.
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)